resumebuilder.client/src/Pages/App.tsx

```diff
@@ -1,27 +1,78 @@
 import React from "react";
 import "./App.scss";
 import "./AntOverrides.scss";
-import { BrowserRouter as Router, Route } from "react-router-dom";
+import { Switch, Route, Redirect } from "react-router-dom";
 import { Layout } from "antd";
 import AppMenu from "./AppMenu/AppMenu";
-import Routes from "../routes.config";
+// components to be routed
+import Home from "./Home/Home";
+import Entry from "./Entry/Entry";
+import Features from "./Features/Features";
 
 const { Header, Content } = Layout;
 
-const App: React.FC = () => {
-    return (
-        <Layout className="AppLayout">
+interface IProps {
+};
+
+interface IState {
+    DisplayMenu: Boolean
+};
+
+class App extends React.Component<IProps, IState> {
+    state: IState;
+
+    constructor(props: IProps) {
+        super(props);
+
+        this.state = {
+            DisplayMenu: true
+        };
+
+        this.handleDisplayMenu = this.handleDisplayMenu.bind(this);
+    }
+
+    private handleDisplayMenu(value: boolean) {
```

Comment:
Better to change this rename to `hideHeader()` and not accept a bool. By default you are already setting DisplayMenu to true.

Reply:
Yes. But I believe when we start navigating programmatically, we will need to reset the state of the pages requiring the header. I didn't get a chance to test it but that is one issue which might happen.